Source/JavaScriptCore/yarr/YarrInterpreter.cpp

```diff
@@ -1287,16 +1461,60 @@ class Interpreter {
         return true;
     }
 
-#define MATCH_NEXT() { ++context->term; goto matchAgain; }
-#define BACKTRACK() { --context->term; goto backtrack; }
+#define MATCH_NEXT() { if (verbose) { dataLog(" - Match\n"); if (btrack) { dataLog("  Matching\n"); btrack = false; } }  ++context->term; goto matchAgain; }
+#define BACKTRACK() { if (verbose) { dataLog(" - Backtrack\n"); if (!btrack) { dataLog("  Backtracking\n"); btrack = true; } } --context->term; goto backtrack; }
```

Comment:
Use `dataLogLnIf` for verbose case.

Reply:
The `verbose` case extends to the next if in both macros.